src/server/game/Spells/SpellEffects.cpp

```diff
@@ -5613,17 +5613,16 @@ void Spell::EffectCreateAreaTrigger(SpellEffIndex /*effIndex*/)
     if (effectHandleMode != SPELL_EFFECT_HANDLE_HIT)
         return;
 
+    Unit* target = unitTarget != nullptr ? unitTarget : GetCaster();
```

Comment:
@joschiwald not really, only unitTarget/gameObjTarget/itemTarget are exclusive with each other, destTarget can be set together with them